tests/test_testing.py

```diff
@@ -266,7 +258,7 @@ def view(company_id):
     with app.test_request_context():
         url = flask.url_for('view', company_id='xxx')
 
-    with app.test_client() as c:
+    with client as c:
```

Comment:
All these can be `with client:` and continue to use `client` in the block. No need to rebind the name in this case.